rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "stable"
```

Comment:
FWIW if your goal is to always have the _latest_ stable automatically (as per recent slack discussion), this probably won't do that, you still have to update your local `stable` via rustup.

Reply:
Ah, yeah, if you leave it on stable, it doesn't. In which case, probably better to have an explicit version

Reply:
I think `channel = "stable"` should be the way to go, like it's done for [`prisma-schema-wasm`](https://github.com/prisma/prisma-engines/blob/43aefb4a29db88d246ea74e047e05571b65aa86f/prisma-schema-wasm/rust-toolchain.toml). Imho, whenever you want to use a new version, it's up to the local dev to update its Rust tools.

1. If you keep `channel = "stable"`, you can simply run `rustup update` and be done with it.
2. If you keep `channel = "1.75.0"` (or similar), you'd need to first update the toolchain file, and then run `rustup update`.

Reply:
Do we generally want to stay on latest as it releases?

Reply:
Usually, yes